docs/book/src/reference/watching-resources/secondary-resources-owned-by-other-controllers.md

```diff
@@ -0,0 +1,78 @@
+# Watching Secondary Resources Owned by Other Controllers or Projects
+
+In some cases, a controller may need to watch and react to changes in resources
+that are not `Owned` by it, but are created and managed by another controller.
+
+The examples below demonstrate how a controller can watch and reconcile resources that it does not directly own.
+This applies to both Kubernetes core API resources (like `ConfigMaps`, `Services`, or `Deployments`) and
+custom resources defined by other projects (such as CRDs from other operators).
+
+For example, if you have two custom resources —`Busybox` and `BackupBusybox`—
+and changes in `Busybox` should trigger a reconciliation for the `BackupBusybox` controller,
+you can configure the `BackupBusybox` controller to watch `Busybox` resources.
+
+### Example: Watching Non-Owned Busybox Resource to Reconcile BackupBusybox
+
+Here's how you can configure the `BackupBusyboxReconciler` to watch changes in the
+`Busybox` resource and trigger reconciliation for `BackupBusybox`:
+
+```go
+// SetupWithManager sets up the controller with the Manager.
+// The controller will watch both the BackupBusybox primary resource and the Busybox resource.
+func (r *BackupBusyboxReconciler) SetupWithManager(mgr ctrl.Manager) error {
+    return ctrl.NewControllerManagedBy(mgr).
+        For(&examplecomv1alpha1.BackupBusybox{}).  // Watch the primary resource (BackupBusybox)
+        Watches(
+            &source.Kind{Type: &examplecomv1alpha1.Busybox{}},  // Watch the Busybox CR
+            handler.EnqueueRequestsFromMapFunc(func(obj client.Object) []reconcile.Request {
+                // Trigger reconciliation for the BackupBusybox in the same namespace
+                return []reconcile.Request{
+                    {
+                        NamespacedName: types.NamespacedName{
+                            Name:      "backupbusybox",  // Reconcile the associated BackupBusybox resource
```

Comment:
In the default scaffold now we add the name. Why?
If we have 2 versions of the same CR such as we have in the tutorial and a controller for each one of them then it will not work out. Also, if we are using the multigroup layout and then, we have the same Kind name and its controllers but for diff groups the same issue will occurs. 

See: https://github.com/kubernetes-sigs/kubebuilder/pull/4162